src/main/java/net/minecraftforge/fml/client/GuiSlotModList.java

```diff
@@ -86,20 +93,24 @@ protected void drawSlot(int idx, int right, int top, int height, Tessellator tes
         if (Loader.instance().getModState(mc) == ModState.DISABLED)
         {
             font.drawString(font.trimStringToWidth(name,       listWidth - 10), this.left + 3 , top +  2, 0xFF2222);
-            font.drawString(font.trimStringToWidth(version,    listWidth - 10), this.left + 3 , top + 12, 0xFF2222);
+            font.drawString(font.trimStringToWidth(version,    listWidth - (4 + height)), this.left + 3 , top + 12, 0xFF2222);
             font.drawString(font.trimStringToWidth("DISABLED", listWidth - 10), this.left + 3 , top + 22, 0xFF2222);
         }
         else
         {
             font.drawString(font.trimStringToWidth(name,    listWidth - 10), this.left + 3 , top +  2, 0xFFFFFF);
-            font.drawString(font.trimStringToWidth(version, listWidth - 10), this.left + 3 , top + 12, 0xCCCCCC);
+            font.drawString(font.trimStringToWidth(version, listWidth - (4 + height)), this.left + 3 , top + 12, 0xCCCCCC);
```

Comment:
It moved the "trimmer" by height from the right edge of the slot. This would create a version-text less square in which the icon lives in the center. But now I changed it to use numbers instead.